tests/integration/deprecated_syscalls_tests.rs

```diff
@@ -106,7 +106,7 @@ async fn test_syscall_get_block_number_cairo0(
         nonce: nonce_manager.next(sender_address),
     });
 
-    let txs = vec![tx];
+    let txs = vec![tx].into_iter().map(Into::into).collect();
```

Comment:
Your way of converting, especially for single txs, is convoluted. You could just do:

let txs = vec![Transaction::AccountTransaction(tx)];`